lib/utils/build-envs.js

```diff
@@ -0,0 +1,22 @@
+import assign from 'lodash/assign';
+
+/**
+ * Returns the default and user specified environment variables.
+ *
+ * @private
+ * @method buildEnvs
+ * @param command {Command} Used to set the default values coming from the specified command
+ * @param [project] {Project} Used to set the environment of the project
+ */
+export default function buildEnvs(command, project) {
```

Comment:
Seems like that would make it available for addon authors, which would be more useful, since putting it on server and test is limited in that sense.